README.md

```diff
@@ -38,11 +38,12 @@ Deploy smart contracts to that local network:
 npx hardhat deploy --network localhost
 ```
 
-The deployed Quiz address will be reported. Remember it and store it
+The deployed Quiz and OasisReward addresses will be reported. Store them
 inside the `frontend` folder's `.env.development`, for example:
 
 ```
 VITE_QUIZ_ADDR=0x385cAE1F3afFC50097Ca33f639184f00856928Ff
+VITE_NFT_ADDR=0x3C8D74Aa9b3390Af534351dB8dcF5A65FE7C9Dd2
```

Comment:
Let's remove VITE_NFT_ADDR and automatically look it up by calling getNftAddress() of the contract.

Reply:
Removed VITE_NFT_ADDR, vue app now fetches from onchain data.